lib/tasks/data_post_processors/case_contact_populator.rb

```diff
@@ -1,6 +1,10 @@
 module CaseContactPopulator
   def self.populate
     CaseContact.find_each do |case_contact|
+      # Get rid of drafts
+      unless case_contact.casa_case
+        case_contact.destroy
+      end
```

Comment:
Not part of the fix, but you have drafts, then running `bin/update` fails without these lines.